src/components/ADempiere/Form/VPOS/ProductInfo/index.vue

```diff
@@ -67,9 +67,18 @@
             </div>
             <div style="width: 30%;float: right;">
               <p style="overflow: hidden;text-overflow: ellipsis;text-align: end;">
-                {{ formatPrice(props.item.priceStandard, props.item.currency.iSOCode) }}
+                {{
+                  formatPrice({
+                    value: props.item.priceStandard,
+                    currencyCode: props.item.currency.iSOCode
+                  })
+                }}
                 <br>
-                {{ formatQuantity(props.item.quantityAvailable) }}
+                {{
```

Comment:
Just can see change for source code with the same functionality, please let it without changes